js/src/ui/TxList/txList.css

```diff
@@ -42,10 +42,11 @@
     }
 
     &.timestamp {
-      padding-top: 1.5em;
-      text-align: right;
+      max-width: 5em;
+      padding-top: 2.25em;
+      text-align: center;
       line-height: 1.5em;
-      opacity: 0.5;
+      color: grey;
```

Comment:
Not 100% sure we use this anywhere, would prefer to keep it where it was to keep consistent (We really need to externalise all colours and just pull in CSS variables - no duplication, consistency is maintained)